### PR TITLE
SQL-1140: Update ODBC Driver name to MONGODB_ODBC_DRIVER

### PR DESCRIPTION
**Pull request by bucaojit**
Updated `ADF_ODBC_DRIVER` to `MONGODB_ODBC_DRIVER`

### CODE REVIEW

File: integration_test/tests/common/mod.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ pub fn generate_default_connection_str() -> String {
     let db = env::var("ADF_TEST_LOCAL_DB");
     let driver = match env::var("ADF_TEST_LOCAL_DRIVER") {
         Ok(val) => val,
-        Err(_e) => "ADF_ODBC_DRIVER".to_string(), //Default driver name
+        Err(_e) => "MONGODB_ODBC_DRIVER".to_string(), //Default driver name
```
Comment (terakilobyte) [inline integration_test/tests/common/mod.rs:17]: Could we move this to constants? I'm working on the setupgui and will be using this a lot. I can do it as part of that PR if you want to leave as is for now.

**Review by terakilobyte — APPROVED** [review]
LGTM, just one comment/question. Waiting for review of the powerbi PR until Alexi confirms.

**Review by terakilobyte — APPROVED** [review]
lgtm

**Review by nbagnard — APPROVED** [review]
LGTM!

**Comment by bucaojit** [comment]
evergreen merge

**Pull request merged**